bemserver_ui/views/timeseries/timeseries_data.py

```diff
@@ -7,6 +7,8 @@
 
 blp = flask.Blueprint("timeseries_data", __name__, url_prefix="/timeseries_data")
 
+periods = {"Year-Monthly": 1, "Year-Daily": 2, "Month-Daily": 3, "Week-Daily": 4, "Week-Hourly":5, "Day-Hourly": 6}
```

Comment:
Probably the definition of "periods" would have been better in [bemserver_ui/static/scripts/modules/views/timeseries/dataQuality.js](https://github.com/BEMServer/bemserver-ui/pull/5/files#diff-f6c89d0cf25dea7d901d85fd765b0aa49641774934c3a2db966ea12cce65c5d5) file.

Reply:
I use this dictionnary to iterate over it in my html file to display it in the completeness page.